delfin/drivers/dell_emc/vmax/alert_handler.py

```diff
@@ -19,23 +19,30 @@
 from delfin import exception
 from delfin.common import constants
 from delfin.drivers.dell_emc.vmax import alert_mapper
+from delfin.drivers.dell_emc.vmax import oid_mapper
 
 LOG = log.getLogger(__name__)
 
 
 class AlertHandler(object):
     """Alert handling functions for vmax driver"""
 
+    def __init__(self):
+        self.oid_mapper = oid_mapper.OidMapper()
+
     # Translation of trap severity to alert model severity
-    SEVERITY_MAP = {"emergency": constants.Severity.FATAL,
-                    "alert": constants.Severity.CRITICAL,
-                    "critical": constants.Severity.CRITICAL,
-                    "error": constants.Severity.MAJOR,
-                    "warning": constants.Severity.WARNING,
-                    "notify": constants.Severity.WARNING,
-                    "info": constants.Severity.INFORMATIONAL,
-                    "debug": constants.Severity.INFORMATIONAL,
-                    "mark": constants.Severity.INFORMATIONAL}
+    # (("unknown", 1), ("emergency", 2), ("alert", 3), ("critical", 4),
+    # ("error", 5), ("warning", 6), ("notify", 7), ("info", 8), ("debug", 9),
+    # ("mark", 10))
```

Comment:
it looks like code comments, plz chnage the style.

Reply:
Updated